aws/tags.go

```diff
@@ -476,3 +476,17 @@ func tagsMapToRaw(m map[string]string) map[string]interface{} {
 
 	return raw
 }
+
+// tagSpecificationsFromMap returns the tag specifications for the given map of data and resource type.
+func tagSpecificationsFromMap(m map[string]interface{}, t string) []*ec2.TagSpecification {
```

Comment:
Since the Go package structure is currently flat across all services, this should include ec2 in the function naming. I'd also recommend clarifying what the input parameters are for Go code tooling.

```go
func ec2TagSpecificationsFromMap(tags map[string]interface{}, resourceType string) []*ec2.TagSpecification {
```